src/main/groovy/org/craftercms/cli/commands/group/CreateGroup.groovy

```diff
@@ -29,14 +29,14 @@ class CreateGroup extends AbstractCommand {
     @CommandLine.Mixin
     GroupOptions groupOptions
 
-    @CommandLine.Option(names = ['-bf', '--bulk-file'],
-            description = 'Absolute path of the CSV file to bulk create groups. CSV file should have the header: name,desc')
-    String bulkFile
+    @CommandLine.Option(names = ['-gi', '--group-import-file'],
+            description = 'Path of Group CSV file to import. The file rows should be: name, description.')
+    String groupFile
 
     @Override
     def run(client) {
-        if (bulkFile) {
-            Paths.get(bulkFile).withReader { reader ->
+        if (groupFile != null && !groupFile.isEmpty()) {
```

Comment:
Regarding this suggestion https://github.com/craftercms/cli/pull/21#discussion_r1613874670, there will be a `NullPointerException` since we are not evaluating the case when not sending the `-gi` command. So we should check first if the `groupFile` is null and then if the `groupFile` is empty.